src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -162,6 +238,14 @@ void MKLDNNFullyConnectedNode::execute(mkldnn::stream strm) {
                     mkldnn::memory::desc newMemDesc(oldMem.get_desc().reshape(normalizedDims));
                     mkldnn::memory newMem(newMemDesc, oldMem.get_engine(), oldMem.get_data_handle());
                     primArgs.at(argType) = newMem;
+                } else {
+                    // in case parameter -> FullyConnected we keep old pointer to data in primArgs on second iteration
```

Comment:
No. This problem have occurred because here we store raw pointer to data in another memory which didn't own it. In other (for parameter -> Node case and dynamism) if we change raw ptr by `set_data_handle` or reallocate, `memory` change `handler` which shared between all memories.